homie/support/repeating_timer.py

```diff
@@ -8,34 +8,25 @@
 
 
 class Repeating_Timer(object):
-    DEFAULT_INTERVAL_REDUCTION = 0.9
```

Comment:
please explain the rationale for removing this

Reply:
Hi, in this case i really did not understand why it's there in the first place. It appeared to be used in the init function as a multiplier of the actual interval provided in the constructor. so it appeared to me that the interval in seconds would have been actually 0.9 x the interval decided by the user with the prior code. If a float was needed (likely) then i would use python conversion of the integer to float. I have also removed the reference to the _time as that would not be needed anymore.